model_templates/inference/python3_sklearn/model-metadata.yaml

```diff
@@ -0,0 +1,10 @@
+name: drumpush-regression
+type: inference
+targetType: regression
+modelID: 5f1f15a4d6111f01cb7f91fd
+environmentID: 5e8c889607389fe0f466c72d
+inferenceModel:
+  targetName: MEDV
+validation:
+  inputData: hi
```

Comment:
Is this expected to be a path?

Reply:
Could you add an example with class labels? Maybe even in regression, but null values.

Reply:
I'm going to follow this up with work to get the validation working.